infrastructure/aws/config.yaml

```diff
@@ -0,0 +1,12 @@
+project_id: "eoapi"
+stage: "dev"
+tags:
+  project: "eoapi"
+public_db_subnet: True # DB in public subnet
+bastion_host: False # no bastion host
+acm_certificate_arn: some_arn # for custom domain names
+stac_api_custom_domain: stac.eoapi.dev # domain name for STAC API
```

Comment:
this is going to create the custom domains

Reply:
🤔 I don't think we want to create the domain, because we could deploy this repo to multiple stage so we don't want to have conflits

Reply:
I can remove that entry, but what kind of conflict are you talking about ? If we want to deploy to another stage we can just change the domain

Reply:
mostly that if you forget to update it by default then you are overriding the production endpoint, maybe we can just have this commented out

Reply:
oh I see what you mean. We could have a config file per stage, that's sort of what we're doing in MAAP.

But I'll turn off the domain for now since it wasn't in your code and just stick to converting what's existing to use the constructs. 

Can always turn on the option later.